.github/workflows/diktat.yml

```diff
@@ -18,4 +18,9 @@ jobs:
           distribution: liberica
       - uses: gradle/gradle-build-action@v2.1.5
         with:
-          arguments: diktatCheckAll --stacktrace
+          arguments: diktatCheckAll -Ddiktat.githubActions=true --stacktrace
+#      - uses: github/codeql-action/upload-sarif@v2
+#        if: ${{ always() }}
+#        with:
+#          sarif_file: /**/build/reports/diktat/diktat.sarif
+#          category: diktat
```

Comment:
Why did you comment it?

Reply:
It doesn't work, Diktat seems to be unable to merge reports from multiple subprojects to one file.

Reply:
Then we should delete it, isn't?

Reply:
Done